app/cafe.py

```diff
@@ -0,0 +1,16 @@
+from datetime import date
+from app import errors
+
+
+class Cafe:
+    def __init__(self, name: int) -> None:
```

Comment:
The `name` parameter should be of type `str`, not `int`. The task description specifies that instances of `Cafe` should have a `name` attribute, which implies it should be a string.